.github/workflows/code_formatting.yml

```
@@ -1,14 +1,26 @@
 name: Check code formatting
 
-on: [push, pull_request]
+on:
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 3: did not find expected key

(syntax-check)